tests/test_file_detection.py

```diff
@@ -0,0 +1,60 @@
+from django.test import TestCase
+
+from multi_import.formats import CsvFormat
+from multi_import.helpers.files import decode_contents
+
+
+class FileDetect(TestCase):
+    def get_file_and_decoded_contents(self, file_path):
+        file = open(file_path, "rb")
+        file.seek(0)
+        file_contents = file.read()
+        decoded_content = decode_contents(file_contents)
+
+        return [file, decoded_content]
```

Comment:
You're missing a `file.close()` call after reading the contents here, but you're also relying on passing the file as the file_handler in the test cases. You could possibly rework this function and the test cases a bit so the file opening/closing is handled via the "with open" style in the test cases below. That will handle closing the file for you after each test run: 

```python
with open("test_file.ext", "rb") as file:
    decoded_contents = get_decoded_contents(file)
    # ...
```